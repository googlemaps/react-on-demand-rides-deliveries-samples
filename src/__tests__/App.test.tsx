import React from 'react';
import { render } from '@testing-library/react-native';

import App from '../../App';

describe('App', () => {
  it('matches previous snapshot', () => {
    const tree = render(<App />);
    expect(tree).toMatchSnapshot();
  });

  it('renders the app corectly', () => {
    const { container } = render(<App />);
    expect(container).toBeTruthy();
  });
});
