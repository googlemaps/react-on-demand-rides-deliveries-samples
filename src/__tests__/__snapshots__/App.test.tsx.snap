// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App matches previous snapshot 1`] = `
<div
  className="css-view-1dbjc4n"
>
  <div
    className="css-view-1dbjc4n"
  >
    <div
      className="css-text-901oao"
      dir="auto"
    >
      LOADING
    </div>
  </div>
  <div
    className="css-view-1dbjc4n r-marginTop-1p6u9xm r-padding-1g4nhe4 r-position-u8s1d r-width-1xce0ei"
  >
    <div
      className="css-view-1dbjc4n"
    >
      <input
        autoCapitalize="sentences"
        autoComplete="on"
        autoCorrect="on"
        className="css-textinput-11aywtz r-borderWidth-rs99b7 r-flex-13awgt0 r-fontSize-1enofrn r-margin-ywje51 r-marginBottom-ku1wi2 r-marginLeft-1cjg8pu r-marginTop-6ity3w r-padding-1ffoksr r-paddingLeft-1hfyk0a r-width-xaq1zp"
        dir="auto"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onSelect={[Function]}
        placeholder="Trip ID"
        readOnly={false}
        spellCheck={true}
        type="text"
      />
      <div
        className="css-view-1dbjc4n r-backgroundColor-1hqxkqz r-borderColor-1atlagf r-borderRadius-1jkafct r-borderWidth-rs99b7 r-cursor-1loqt21 r-marginLeft-1cjg8pu r-padding-tuq35u r-touchAction-1otgn73 r-transitionProperty-1i6wzkk r-userSelect-lrvibr r-width-1j6l8hp"
        onClick={[Function]}
        onContextMenu={[Function]}
        onKeyDown={[Function]}
        style={
          Object {
            "WebkitTransitionDuration": "0s",
            "transitionDuration": "0s",
          }
        }
        tabIndex="0"
      >
        <div
          className="css-text-901oao r-color-jwli3a r-fontSize-1enofrn r-textAlign-q4m81j"
          dir="auto"
        >
          Find
        </div>
      </div>
    </div>
  </div>
</div>
`;
